heartbeat/monitors/util.go

```diff
@@ -53,16 +52,16 @@ func (s IPSettings) Network() string {
 	return ""
 }
 
-func MakeSimpleJob(name, typ string, f func() (common.MapStr, error)) Job {
-	return MakeJob(name, typ, func() (common.MapStr, []TaskRunner, error) {
+func MakeSimpleJob(jobName string, f func() (common.MapStr, error)) Job {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function MakeSimpleJob should have comment or be unexported